docs/build_docs

```diff
@@ -30,19 +30,9 @@ for version in $(jq -r '.[]' $VERSIONS_JSON_PATH); do
 
 done
 
-# Get latest released version from tag, check out to build API docs.
-# Only if not PR deploy preview.
-if [ "$PULL_REQUEST" == "false" ]
-then
-  GX_LATEST=$(git tag | grep -E "(^[0-9]{1,}\.)+[0-9]{1,}" | sort -V | tail -1)
-  echo -e "${ORANGE}Not in a pull request. Using latest released version ${GX_LATEST} at $(git rev-parse HEAD) to build API docs.${NC}"
-  git checkout "$GX_LATEST"
-  git pull
-else
-  echo -e "${ORANGE}Building from within a pull request, using the latest commit to build API docs so changes can be viewed in the Netlify deploy preview.${NC}"
-  git checkout "$CURRENT_COMMIT"
-  git pull
-fi
+echo -e "${ORANGE}Prepare prior versions using the current commit (e.g. proposed commit if in a PR or develop if not).${NC}"
```

Comment:
What do `ORANGE` and `NC` represent?

Reply:
Those are ANSI escape codes defined earlier in the file, just so it's a little easier to read the echo statements:
```
ORANGE='\033[0;33m'
NC='\033[0m' # No Color
```
 These change the text to orange to more easily read the netlify deploy logs.